13_factorial/README.md

```diff
@@ -0,0 +1,12 @@
+# Exercise 13 - Factorial
+
+Write a [recursive factorial function](https://simple.wikipedia.org/wiki/Factorial) that takes a non-negative integer, and returns the product of all positive integers less than or equal to the input integer. An input of `0` should return `1`. The function should only accept numbers, so `'4'` should not be accepted as it is a string. All invalid inputs should return `undefined`.
```

Comment:
```suggestion
Write a recursive [factorial](https://simple.wikipedia.org/wiki/Factorial) function that takes a non-negative integer, and returns the product of all positive integers less than or equal to the input integer. An input of `0` should return `1`. The function should only accept numbers, so `'4'` should not be accepted as it is a string. All invalid inputs should return `undefined`.
```
Nit: Though the article does mention recursion, it's not about recursion nor about programming functions, only "factorial" itself so the link text ought to reflect that exactly.

(**love** simple wiki!)